webpack.config.cjs

```diff
@@ -72,13 +72,15 @@ const sharedConfig = {
       !isProduction && {
         test: /\.m?js$/,
         use: ['source-map-loader'],
+        exclude: /node_modules/,
```

Comment:
Was seeing [lots of warnings in the webpack output](https://gist.github.com/samwhale/c0d6d94711332200fd170423d75c69cb) when the source-map-loader is running through the node modules. Seeing these warnings in main too. Can remove these by excluding node modules.

🤔 However, I think this is a bad way to approach fixing these errors. It should be helpful to have the source map loader for 3rd party packages with their own source maps defined, right?

Reply:
Instead of just excluding the whole of `node_modules` it would be better to just exclude the offending modules and write a comment explaining the config.

> It should be helpful to have the source map loader for 3rd party packages with their own source maps defined, right?

Theoretically yes, but the warnings mean that packages like `blurhash` are not shipping with source maps. This is actually fixed in the latest release of `blurhash`, see https://github.com/woltapp/blurhash/pull/155

The same goes for `html-to-image` and `react-blurhash`, according to that log.

Reply:
Fixed in b449e83c760952835809a487fbe4fe39cb432764 and 3a513537a230c32197499189600f6a1c6290539a